src/extensions/FocusTrap.js

```diff
@@ -0,0 +1,41 @@
+import { Extension } from '@tiptap/core'
+
+const toggleFocusTrap = ({ editor }) => {
+	const trapStack = window._nc_focus_trap ?? []
+	const activeTrap = trapStack[trapStack.length - 1]
+
+	const possibleEditorTabCommand = editor.can().sinkListItem('listItem')
```

Comment:
This will throw an exception when `listItem` is not a registered node type:

`Error: There is no node type named 'listItem'. Maybe you forgot to add the extension?`

We probably either have to catch the exception or check if the node type exists first.

Reply:
Fixed by only using it for the rich text editing. Plaintext has a different problem, but I'll address that separately.